config/src/config.rs

```diff
@@ -483,6 +483,10 @@ pub struct Config {
     #[dynamic(default)]
     pub background: Vec<BackgroundLayer>,
 
+    /// Only works on MacOS
+    #[dynamic(default)]
+    pub window_background_blur: i64,
```

Comment:
naming wise: if this can only every apply to macos, then it would be preferable to rename it to `macos_window_background_blur`.

If it is feasible to also make this same setting work with the same semantics on Windows, which I believe has a similar concept, then keeping the name more general is reasonable.

Reply:
OK, I rename it to `macos_window_background_blur`
And on Windows seems to need to use the gaussian blur effect